File: src/main/java/com/github/frcsty/frozenactions/actions/player/ActionbarMessageAction.java
package com.github.frcsty.frozenactions.actions.player;

import com.github.frcsty.frozenactions.actions.Action;
import com.github.frcsty.frozenactions.actions.ActionContext;
import org.jetbrains.annotations.NotNull;

public final class ActionbarMessageAction implements Action {

    @Override
    public @NotNull String getId() {
        return "ACTIONBARMESSAGE";
    }

    @Override
    public void run(final @NotNull ActionContext context) {
        context.getPlayer().sendActionBar(context.getData());
    }
}

File: src/main/kotlin/com/github/frcsty/frozenactions/actions/player/ActionbarMessageAction.kt
package com.github.frcsty.frozenactions.actions.player

import com.github.frcsty.frozenactions.actions.Action
import com.github.frcsty.frozenactions.util.sendActionBarMessage
import org.bukkit.entity.Player

object ActionbarMessageAction : Action {
    override val id = "ACTIONBARMESSAGE"

    override fun run(player: Player, data: String) {
        player.sendActionBarMessage(data)
    }
}